src/viur/core/tasks.py

```diff
@@ -655,7 +655,11 @@ def mkDecorator(fn):
                                f"Please rename {fn.__name__!r}")
         if cronName not in _periodicTasks:
             _periodicTasks[cronName] = {}
-        _periodicTasks[cronName][fn] = interval
+        if isinstance(interval, datetime.timedelta):
+            # convert seconds to minutes
+            _periodicTasks[cronName][fn] = int(interval.total_seconds() / 60)
+        else:
+            _periodicTasks[cronName][fn] = int(interval)
```

Comment:
ld be of course a breaking change.

1. Can we save the `timedelta` instance and remove [this `timedelta` creation here](https://github.com/viur-framework/viur-core/blob/45ca4b37767645bff3a09efe20f3580150d39d83/src/viur/core/tasks.py#L265C69-L265C105)?
2. I've implemented this method a few weeks ago: [`utils.parse.timedelta`](https://github.com/viur-framework/viur-core/blob/47e6955c62e715297274b7541409f7ccba34d747/src/viur/core/utils/parse.py#L23-L36). My intention was to avoid if/else's like this, but still support both ways. Unfortunately, we have minutes here, not seconds. But this is the next problem, the inconsistent use of int values with different meanings, we should normalize this. But of course that would be a breaking change -- for me this would be okay.

Reply:
> 1. Can we save the `timedelta` instance and remove [this `timedelta` creation here](https://github.com/viur-framework/viur-core/blob/45ca4b37767645bff3a09efe20f3580150d39d83/src/viur/core/tasks.py#L265C69-L265C105)?

Yes, that would be better.

> 2. I've implemented this method a few weeks ago: [`utils.parse.timedelta`](https://github.com/viur-framework/viur-core/blob/47e6955c62e715297274b7541409f7ccba34d747/src/viur/core/utils/parse.py#L23-L36). My intention was to avoid if/else's like this, but still support both ways. Unfortunately, we have minutes here, not seconds. But this is the next problem, the inconsistent use of int values with different meanings, we should normalize this. But of course that would be a breaking change — for me this would be okay.

Yes, please. We can consider about adding a "compatiblity" mode for 3.x first, which still accepts ints and throws a big-fat warning.

```suggestion
        if isinstance(interval, int):
            logging.warning(
                f"Assuming {interval=} minutes here. This will change into seconds in future. Please use `datetime.timedelta` for clarification."
             )
             interval *= 60

        _periodicTasks[cronName][fn] = utils.parse.timedelta(interval)
```